src/app/(project)/projects/[project]/collections/page.tsx

```diff
@@ -0,0 +1,216 @@
+                      <span className="rounded-full h-4 w-28 bg-stone-800 animate-pulse" />
+                    </div>
+                  ),
+                  cell: () => (
+                    <div className="flex gap-2 items-center">
+                      <span className="rounded-md h-8 w-8 bg-stone-800 animate-pulse" />
+                      <span className="rounded-full h-4 w-28 bg-stone-800 animate-pulse" />
+                    </div>
+                  ),
+                }),
+                loadingColumnHelper.display({
+                  id: 'nfts',
+                  header: () => (
+                    <div className="flex gap-2 items-center">
+                      <span className="rounded-full h-4 w-28 bg-stone-800 animate-pulse" />
+                    </div>
+                  ),
+                  cell: () => (
+                    <div className="flex items-center">
+                      <span className="rounded-full h-4 w-28 bg-stone-800 animate-pulse" />
+                    </div>
+                  ),
+                }),
+                loadingColumnHelper.display({
+                  id: 'type',
+                  header: () => (
+                    <div className="flex items-center">
+                      <span className="rounded-full h-4 w-28 bg-stone-800 animate-pulse" />
+                    </div>
+                  ),
+                  cell: () => (
+                    <div className="flex items-center">
+                      <span className="rounded-full h-4 w-28 bg-stone-800 animate-pulse" />
+                    </div>
+                  ),
+                }),
+                loadingColumnHelper.display({
+                  id: 'createdAt',
+                  header: () => (
+                    <div className="rounded-full h-4 w-28 bg-stone-800 animate-pulse" />
+                  ),
+                  cell: () => (
+                    <div className="flex flex-col gap-1">
+                      <span className="rounded-full h-3 w-16 bg-stone-800 animate-pulse" />
+                      <span className="rounded-full h-3 w-8 bg-stone-800 animate-pulse" />
+                    </div>
+                  ),
+                }),
+                loadingColumnHelper.display({
+                  id: 'options',
+                  meta: {
+                    align: 'right',
+                  },
+                  header: () => <div className="rounded-full h-4 w-4 bg-stone-800 animate-pulse" />,
+                  cell: () => <div className="rounded-full h-4 w-4 bg-stone-800 animate-pulse" />,
+                }),
+              ]}
+              data={new Array(4)}
+            />
+          </>
+        ) : (
+          <>
+            <h1 className="text-2xl text-white font-medium">Collections</h1>
+            {noCollections ? (
+              <div className="h-full flex-1 flex flex-col items-center justify-center">
+                <Icon.Large.CreateProject />
```

Comment:
I don't think this is the write icon. Maybe set to the same one we use when there is no mints.

@damianr what should the empty state look like for collections?

Reply:
There is no specific icon for empty mints as well.